website/server/models/user/methods.js

```diff
@@ -4,8 +4,7 @@ import {
   chatDefaults,
   TAVERN_ID,
 } from '../group';
-import { defaults } from 'lodash';
-
+import { defaults, map, flatten, flow, compact, unique, partialRight } from 'lodash';
```

Comment:
Heads up, we're migrating to lodash v4 (https://github.com/HabitRPG/habitica/pull/8495) which includes many breaking changes (https://github.com/lodash/lodash/wiki/Changelog#v400). Before merging this PR should be checked and updated to make sure all lodash methods are compatible with v4.